src/components/LoginRegister/SignUp.js

```diff
@@ -16,6 +16,7 @@ export default function Login() {
     if(login) history.push("/app");
   }, [login, history])
 
+  // comment on naming and extraction
   const fetchUser = () => {
```

Comment:
really bad naming here haha. this function is registering a user - it's name should say that. Same thing with the "put this in a separate utility" file as i explained above, for all the same reasons